common-ktx/src/main/java/co/nimblehq/common/extensions/TypedArrayExt.kt

```diff
@@ -0,0 +1,92 @@
+package co.nimblehq.common.extensions
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.content.res.TypedArray
+import android.graphics.Typeface
+import android.graphics.drawable.Drawable
+import android.os.Build
+import androidx.annotation.FontRes
+import androidx.annotation.StyleableRes
+import androidx.appcompat.content.res.AppCompatResources
+import androidx.core.content.res.ResourcesCompat
+
+/**
+ * getTypeface from Compat first, if could nof found, then try to create with the family name
+ *
+ * @param context - Android context
+ * @param index -  StyleableRes index, ex: R.styleable.your_custom_attribute
```

Comment:
```suggestion
 * @param index - StyleableRes index, sample: R.styleable.your_custom_attribute
```
I'm not sure should we use the `sample` here instead of `ex`? What do you guys think? @doannimble @hoangnguyen92dn

Reply:
@lydiasama I'm good with both

Reply:
I think we should use `sample` for consistency 😄